Plant_tracker/Plant_tracker/src/main/java/com/example/Plant_tracker/Plant.java

```diff
@@ -0,0 +1,8 @@
+import jakarta.persistence.*;
+
+@Entity
+@Table(name = plants)
+public class Plant {
+    private int id;
```

Comment:
Czy to pole sie liczy jako unique?

Reply:
w sumie mogę ten zewnętrzny folder nazwać backend. Byłoby ok?

Reply:
Powiem, że jeszcze nie jestem pewna, co robią te adnotacje i jak działają, ale id będzie miało być unique